### ISSUE TITLE
Death Talon Wyrmkin, Unstunnable and not interruptible.

### ISSUE THREAD

**Issue by Meatsack** [elysium-project/server#2397]
--
I just wanted to adress the current situation on Elysium inside Blackwing lair. The trash after the boss Vaelastrasz the Corrupt aren't working properly due to their immunity to both stun effect and interrupt abilties. By just watching some old vidoes on www.warcraftmovies.com from blackwing lair runs you can see people constantly interrupting their abilities. Leaving them for last and just ccing them to death. Since warcraft movies only allows downloads through subscribing to them I can't really link them as evidence but I'm sure elysium staff have their own access to it so they can just confirm it.

**Comment by Cabro** [comment]
Scorching Elemental has the same godlike features. It's a normal mob in Un'Goro...

**Comment by BrynjaDarrow** [comment]
Druids can sleep them, right?

**Comment by Meatsack** [comment]
Yes sleep is the only thing that works properly, But it seems that you can't even spam sleep on them to interupt their casts. Its super wierd all those old school tactics have 0 effect. they just blast everything apart

**Comment by GuybrushGit** [comment]
Link to video + timestamp.

**Comment by thepparker** [comment]
As Guybrush said, can you link any videos where it is clearly evident?
